modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -46,45 +58,47 @@ The repositories and channels for synchronizing upstream content must be configu
 In the {productname} {webui}, navigate to menu:Software[Channel List > All].
 Verify that you have a base channel and a child channel for your architecture.
 
-For example, for {ubuntu} 18.04:
-
-* Base channel: [systemitem]``ubuntu-18.04-pool for amd64``
-* Child channel: [systemitem]``Ubuntu-18.04-SUSE-Manager-Tools for amd64``
+For example, for {ubuntu} 20.04:
```

Comment:
I don't understand, why do you have this if you already explained how to add the channels above?

Reply:
This is a verification step.

Reply:
Can you please generate the rendered doc? At least on the adoc this looks wrong.